src/service/comment.js

```diff
@@ -84,6 +84,7 @@ const createChildComment = async (
     if (isAnonymous) {
       data = {...data, anon_user_info: anonimity};
     }
+    console.log(data, 'nusuk');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It's not a good practice to leave `console.log` statements in the production code. It can lead to potential security risks as sensitive information might get logged. Also, it can clutter the console output making it hard to read important logs. Consider removing this line or replace it with a proper logging mechanism.
```diff
-    console.log(data, 'nusuk');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->